plugins/com.google.cloud.tools.eclipse.util/templates/appengine/MockHttpServletResponse.java.ftl

```diff
@@ -0,0 +1,174 @@
+<#if package != "">package ${package};
+
+</#if>import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Locale;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This class is created only for the purpose of demonstrating unit testing of the
+ * {@link HelloAppEngine} class and the only those methods have meaningful implementation
+ * which are use in the unit test.
```

Comment:
I guess what you tried to say is

"Only those methods that have meaningful implementation are used in the unit test."

or

"Only those methods that are used in the unit test have meaningful implementation."

I think it's also good to cut this long statement into two, e.g., `... HelloAppEngine} class. (period) Only those ...`